crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -279,18 +279,18 @@ impl RealRibosome {
                             let i: u8 = i
                                 .try_into()
                                 .map_err(|_| ZomeTypesError::EntryTypeIndexOverflow)?;
-                            EntryDefIndex(i)
+                            i
```

Comment:
We're getting the total number of entry defs here, not a specific entry def. This thoroughly confused me as I suddenly questioned my understanding what an EntryDefIndex is here for. And I see no need to construct an `EntryDefIndex` here if we're only interested in the total number for the purpose of generating the `GlobalZomeTypes` type further down.